src/core/stateManager/State.cpp

```diff
@@ -0,0 +1,21 @@
+#include "State.hpp"
+namespace Yuna
+{
+	namespace Core
+	{
+		State::State(/* args */)
+		{
+
+		}
+
+		State::~State()
+		{
+
+		}
+
+		void	State::Run()
+		{
+			//Main Loop //MATHIAS
```

Comment:
You can now merge your feature branch with dev (if pull request is accepted) to start implementing the game loop.